nemo/collections/asr/parts/numba/rnnt_loss/utils/cuda_utils/gpu_rnnt_kernel.py

```diff
@@ -961,7 +961,9 @@ def compute_tdt_alphas_kernel(
             if t > 0 and t < T:
                 alphas[offset + t * maxU + u] = -INF
 
-                for i in range(1, num_durations):  # skip 0 since blank emission has to advance by at least one
+                for i in range(0, num_durations):
```

Comment:
just make it single variable range. No need to make it overt

Reply:
Done.